extensions/wikia/ArticleVideo/scripts/article-video.js

```diff
@@ -98,36 +83,25 @@ require(['wikia.window', 'wikia.onScroll', 'wikia.tracker', 'ooyalaVideo'], func
 			});
 		}
 
-		function updatePlayerControls(waitForTransition) {
-			ooyalaVideoController.hideControls();
-			if (waitForTransition && transitionEndEventName) {
-				$videoContainer.on(transitionEndEventName, function () {
-					ooyalaVideoController.sizeChanged();
-					ooyalaVideoController.showControls();
-				});
-			} else {
-				ooyalaVideoController.sizeChanged();
+		function updateOoyalaSize() {
+			window.dispatchEvent(new Event('resize'));
+			// wait for player resize - there is 150ms debounce on resize event in ooyala html5-skin
+			setTimeout(function () {
 				ooyalaVideoController.showControls();
-			}
+			}, 150);
 		}
 
-		function isVideoPlaying() {
-			if (ooyalaVideoController && ooyalaVideoController.player) {
-				return ooyalaVideoController.player.getState() === OO.STATE.PLAYING;
-			}
-			return false;
-		}
-
-		function isVideoPausedOrReady() {
-			if (ooyalaVideoController && ooyalaVideoController.player) {
-				return ooyalaVideoController.player.getState() === OO.STATE.PAUSED || ooyalaVideoController.player.getState() === OO.STATE.READY;
+		function updatePlayerControls(waitForTransition) {
+			ooyalaVideoController.hideControls();
+			if (!waitForTransition) {
+				updateOoyalaSize();
 			}
-			return false;
+			// otherwise wait for SIZE_CHANGED event and then execute updateOoyalaSize function
```

Comment:
It's not really _otherwise_, right? When `waitForTransition === false`, we will call `updateOoyalaSize()` twice - in the `if` block above and on `OO.EVENTS.SIZE_CHANGED`. Are we fine with it?